app/controllers/rooms_controller.rb

```diff
@@ -84,25 +85,24 @@ def destroy
   # GET /launch.json?
   def launch
     redirector = room_path(@room.id)
-    redirect_to redirector
+    redirect_to(redirector)
```

Comment:
This was only for debugging, you can change it to

    redirect_to(room_path(@room.id))